multi-buffer-search/src/commands/replaceCommand.ts

```
@@ -0,0 +1,129 @@
+import { ReplaceOptions, MultiBufferDocument } from '../models/types'
+import { MultiBufferProvider } from '../multiBufferProvider'
+
+export class ReplaceCommand extends SearchCommand {
+  constructor(
+    searchService: SearchService,
+    formatter: ResultFormatter,
+    decorationService: DecorationService,
+    multiBufferProvider: MultiBufferProvider
+  ) {
+    super(searchService, formatter, decorationService, multiBufferProvider)
+  }
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)